lib/rubocop/cop/style/redundant_parentheses.rb

```diff
@@ -81,6 +87,14 @@ def method_call_with_redundant_parentheses?(node)
 
           args.empty? || parentheses?(send_node) || square_brackets?(send_node)
         end
+
+        def first_argument?(node)
```

Comment:
I wonder if we're not using a similar method in other cops. Seems generic enough.